phpstan.neon

```diff
@@ -1,6 +1,8 @@
 parameters:
   level: 9
   phpVersion: 70430 # PHP 7.4.30
+  typeAliases:
+    IntervalStatus: 'array{0:string, 1:boolean}'
```

Comment:
@staabm I found that I could not use a local type alias because of:
1) https://github.com/phpstan/phpstan/issues/9424 - local type alias cannot be the return type of a function
2) I couldn't get it to work in https://github.com/sabre-io/event/blob/master/lib/Loop/functions.php - that code is not a class, and `@phpstan-import-type` does not seem to be effective for "ordinary" code that is not a class.

So I made a global type alias. That works everywhere.

But will doing that be an issue for consumers of this? They will use PHPstan to analyse their use of `setInterval` and `clearInterval`, but their `phpstan.neon` will not know what `IntervalStatus` is. They might have to add this to their `phpstan.neon`

Reply:
https://github.com/phpstan/phpstan/issues/10154

Reply:
The older issue is https://github.com/phpstan/phpstan/issues/9164

Reply:
the global type aliases will not work for library consumers.
(or only if they also define this alias in their config).

not sure this is worth it since the functions/methods involved with this type are the actual api surface of our package. I think we should stop using it until phpstan supports using the local alias properly for functions.

Reply:
I made PR #116 that has the code from here, without the last commit. Please review that. Then I can close this PR #114 - leaving this discussion.